src/components/voting/votingBar.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+
+import votingConst from '../../constants/voting';
+import style from './votingBar.css';
+
+const VotingBar = ({ votes }) => {
+  const { maxCountOfVotes, maxCountOfVotesInOneTurn } = votingConst;
+  const votedList = Object.keys(votes).filter(key => votes[key].confirmed);
```

Comment:
Please use a single `forEach` loop instead of 3 `filter`s and file the lists thought the same iteration

Reply:
I set up a jsperf to see the difference:
https://jsperf.com/multiple-filters-vs-foreach

ForEach is 3 or 4 times faster, but the absolute values of how long it takes on a list of 100 still make the difference negligible.

I like using `fliter`s for their conciseness and readability.